rubocop.yml

```diff
@@ -171,5 +174,13 @@ Style/ExponentialNotation:
 Style/SlicingWithRange:
   Enabled: true
 
+# Re-enable this when the following is resolved:
+# https://github.com/rubocop-hq/rubocop/issues/8098
+Style/RedundantRegexpCharacterClass:
+  Enabled: false
+
+Style/RedundantRegexpEscape:
```

Comment:
Conflicted about this, it will raise a lot of warnings that need to be fixed manually...

Reply:
I think there's an autocorrect for this...